drivers/bridge/bridge.go

```diff
@@ -168,9 +168,23 @@ func (c *networkConfiguration) Conflict(o *networkConfiguration) bool {
 	return false
 }
 
+func (n *bridgeNetwork) isDefaultNetwork() bool {
+	if n.config.BridgeName == DefaultBridgeName {
+		return true
+	}
+	return false
+}
+
 // FromMap retrieve the configuration data from the map form.
-func (c *networkConfiguration) FromMap(data map[string]interface{}) error {
+func (c *networkConfiguration) FromMap(d *driver, data map[string]interface{}) error {
```

Comment:
I think we could have the caller just pass the default config t othe parsing function and then down to here.
it just look weird passing the driver pointer down to here and make the other method driver methods (for the coking issue)
We could just look for the default network in Createnetwork().

Not a big deal, we can take care of this later. Up to you. Therefore, 
LGTM.